parcels/field.py

```diff
@@ -588,6 +589,7 @@ def interpolator2D(self, ti, z, y, x):
         xi = 0
         yi = 0
         (xsi, eta, _, xi, yi, _) = self.search_indices(x, y, z, xi, yi)
+        return self.data[ti, yi+1, xi+1]
```

Comment:
Why `return` here? Shouldn't we instead do an `assert` of the indices themselves?

Reply:
Woups! This is a completely unrelated line that shouldn't have been commited...